src/components/Facets.tsx

```diff
@@ -0,0 +1,80 @@
+import { Facets as FacetsCompoundComponent } from './Filters/Facets';
+import { FilterGroup, FilterGroupCssClasses } from './FilterGroup';
+import { CompositionMethod } from '../hooks/useComposedCssClasses';
+import { Fragment } from 'react';
+
+/**
+ * The CSS class interface for {@link Facets}.
+ *
+ * @public
+ */
+export interface FacetsCssClasses extends FilterGroupCssClasses {
+  container?: string,
+  divider?: string
+}
+
+/**
+ * Props for the {@link Facets} component.
+ *
+ * @public
+ */
+export interface FacetsProps {
+  /** {@inheritDoc FilterGroupProps.collapsible} */
+  collapsible?: boolean,
+  /** {@inheritDoc FilterGroupProps.defaultExpanded} */
+  defaultExpanded?: boolean,
+  /** {@inheritDoc FilterGroupProps.searchable} */
+  searchable?: boolean,
+  /**
+   * Whether or not a search is automatically run when a filter is selected.
+   * Defaults to true.
+   */
+  searchOnChange?: boolean,
+  /** CSS classes for customizing the component styling. */
+  customCssClasses?: FacetsCssClasses,
+  /** {@inheritDoc CompositionMethod} */
+  cssCompositionMethod?: CompositionMethod
+}
+
+/**
+ * A component that displays all facets applicable to the current vertical search.
+ *
+ * @param props - {@link FacetsProps}
+ * @returns A React component for facets
+ *
+ * @public
+ */
+export function Facets(props: FacetsProps) {
+  const { searchOnChange, customCssClasses = {}, ...filterGroupProps } = props;
+  const { container:containerCssClasses, divider, ...filterGroupCssClasses } = customCssClasses;
+  return (
+    <FacetsCompoundComponent searchOnChange={searchOnChange} className={containerCssClasses}>
+      {facets => {
+        const filteredFacets = facets.filter(f => f.options.length > 0);
```

Comment:
Is the plan right now to push HierarchicalFacets off to a later date, and for now hierarchical facets will just get rendered as regular facets?

Reply:
the item mentioned to excludes Hierarchical & Numerical Facets. I imagined we will schedule work for it later

Reply:
if this component should exclude Numerical Facets, should we only support primitive value types and basic matchers?

Reply:
per all hands discussion and confirmation with product, we will be including support for numerical and hierarchical facets in the same component so the interface should remain the same